docs/config/README.md

```diff
@@ -1729,6 +1729,39 @@ pyenv_version_name = true
 pyenv_prefix = "foo "
 ```
 
+## R
+
+The `r` module shows the currently installed version of R.
+The module will be shown if any of the following conditions are met:
+
+- The current directory contains a `.R` file
```

Comment:
I believe you should check for `.Rproj` files too.

Reply:
Just added this file type. Thanks for the review and sorry for the delay!